### PR TITLE
Delete extra modal when deleting running container

### PR DESCRIPTION
**Pull request by QiWang19**

### CODE REVIEW

**Comment by QiWang19** [comment]
@martinpitt PTAL

File: src/Containers.jsx [hunk 1]
```diff
@@ -33,11 +33,17 @@ class Containers extends React.Component {
     }
 
     deleteContainer(container, event) {
-        event.preventDefault();
```
Comment (martinpitt) [inline src/Containers.jsx]: It's good that this gets dropped (it should rarely be used), but just want to ensure that this is deliberate?

File: src/Containers.jsx [hunk 2]
```diff
@@ -124,7 +130,7 @@ class Containers extends React.Component {
     }
 
     handleRemoveContainer() {
-        const container = this.state.containerWillDelete;
+        // const container = this.state.containerWillDelete;
```
Comment (martinpitt) [inline src/Containers.jsx]: Please just remove obsolete code instead of commenting it out.

File: test/check-application [hunk 3]
```diff
@@ -74,7 +74,7 @@ class TestApplication(testlib.MachineCase):
 
         # delete running container busybox using force delete
         b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
-        self.confirm_modal("btn-ctr-delete")
+        # self.confirm_modal("btn-ctr-delete")
```
Comment (martinpitt) [inline test/check-application]: dito.

File: src/Containers.jsx (hunk 1 above)
Comment (QiWang19) [reply src/Containers.jsx]: Yes. I  deleted it deliberately.

**Comment by QiWang19** [comment]
@martinpitt PTAL

**Comment by martinpitt** [comment]
BTW, please refer to issues that a PR fixes with `Fixes #123` in the commit log. I'll do it on merging now, but it's good practice to train yourself to do that.

**Comment by QiWang19** [comment]
:ok_hand:  Thanks for telling me this.

**Pull request merged**